.github/workflows/documentation.yml

```diff
@@ -0,0 +1,26 @@
+name: Docs
+on:
+  push:
+    branches: [aci.documentation]
```

Comment:
Will this eventually be carried over to the Credentials project for the `master` branch? Or is this just temporary for this feature branch?

Reply:
Nope. It's temporary, just for internal convenience.